adcpipeline/pipeline.py

```diff
@@ -100,12 +99,14 @@ def from_yaml_file(cls, df: DataFrame, path: str):
         Args:
             df: This is your data in a DataFrame format.
             path: Path to yaml file.
+            filename: Path/filename for caching
 
         Returns:
             Instance of this class.
         """
-        settings = LoadConfig.load_yaml_as_dict(path)['pipeline']
-        return cls(df=df, method_settings=settings)
+        with open(file=path, mode='r') as f:
```

Comment:
Shouldn't we check first if the path exists?

Reply:
If you call `from_yaml_file` factory method, I assume you know you are trying to read a file. If it then can't find it, I don't mind its throwing just the regular ole error of not finding the file :)